batchflow/decorators.py

```diff
@@ -82,6 +82,15 @@ def another_critical_section(self, some_arg, another_arg):
     # action with arguments
     return _make_action_wrapper_with_args(*args, **kwargs)
 
+def apply_transform(method):
```

Comment:
Now it works only as a marker for meta-class. Make it fully functional so it can wrap methods without meta-class